server/request_object.cc

```diff
@@ -126,7 +126,11 @@ RequestObject::RequestObject(
 
   S3Uuid uuid;
   request_id = uuid.get_string_uuid();
-  s3_log(S3_LOG_DEBUG, request_id, "Constructor\n");
+  // last 12 characters
+  stripped_request_id = std::string(request_id.end() - 12, request_id.end());
```

Comment:
write a unit test to test that the stripped_request_id will always have 12 chars.

Reply:
Done